dashboard/test/lib/pd/payment/payment_calculator_counselor_admin_test.rb

```diff
@@ -6,19 +6,14 @@ class PaymentCalculatorCounselorAdminTest < ActiveSupport::TestCase
       # TIME_CONSTRAINTS_BY_SUBJECT: SUBJECT_ECS_PHASE_4 => {min_days: 2, max_days: 3, max_hours: 18}
       @workshop = create :pd_ended_workshop,
         workshop_type: Pd::Workshop::TYPE_PUBLIC,
-        course: Pd::Workshop::COURSE_ECS,
-        subject: Pd::Workshop::SUBJECT_ECS_PHASE_4,
+        course: Pd::Workshop::COURSE_COUNSELOR,
         num_sessions: 3
 
       # 2 facilitators
       2.times do
         @workshop.facilitators << create(:facilitator)
       end
 
-      # One unqualified teacher, below min attendance
-      create :pd_workshop_participant, workshop: @workshop,
-        enrolled: true, in_section: true, attended: [@workshop.sessions.first]
-
```

Comment:
CounselorAdmin workshops use the [default](https://github.com/code-dot-org/code-dot-org/blob/322693ac6ab42c929d0a4649dbb4660d11ff225d/dashboard/app/models/pd/workshop.rb#L362) 1 minimum session, so all this testing of below min attendance is irrelevant here now that we're actually using the correct workshop course.